web/src/lib/utils.ts

```diff
@@ -269,12 +269,14 @@ export const oauth = {
   isAutoLaunchDisabled: (location: Location) => {
     const values = ['autoLaunch=0', 'password=1', 'password=true'];
     for (const value of values) {
-      if (location.search.includes(value)) {
-        return true;
-      }
+      if (location.search.includes(value)) return true;
     }
     return false;
   },
+  isAutoLaunchEnabled: (location: Location) => {
+    const value = 'autoLaunch=1';
+    return (location.search.includes(value));
```

Comment:
I feel like those braces around `location.search.includes` will be prettier warning.